src/components/themed/WalletListCurrencyRow.js

```diff
@@ -54,10 +55,16 @@ class WalletListRowComponent extends React.PureComponent<Props> {
 
     // Yesterdays Percentage Difference
     const yesterdayUsdExchangeRate = exchangeRates[`${currencyCode}_iso:USD_${getYesterdayDateRoundDownHour()}`]
-    const yesterdayExchangeRate = yesterdayUsdExchangeRate * fiatExchangeRate
-    const differenceYesterday = exchangeRate ? exchangeRate - yesterdayExchangeRate : null
+    const yesterdayExchangeRate =
+      yesterdayUsdExchangeRate != null && fiatExchangeRate != null ? bns.mul(yesterdayUsdExchangeRate.toString(), fiatExchangeRate.toString()) : '0'
+
+    const differenceYesterday = exchangeRate ? bns.sub(exchangeRate.toString(), yesterdayExchangeRate) : null
+
+    let differencePercentage =
+      differenceYesterday && yesterdayExchangeRate !== '0'
+        ? bns.mul((parseFloat(differenceYesterday) / parseFloat(yesterdayExchangeRate)).toString(), '100')
```

Comment:
Don't use parseFloat, just stay in strings and use bns

Reply:
Just multiply by a 100 x 10^(num of decimal places)